recipe/rsync.php

```diff
@@ -137,8 +137,8 @@
     $server = $server->getConfiguration();
     $host = $server->getHost();
     $port = $server->getPort() ? ' -p' . $server->getPort() : '';
-    $identityMethod = $server->getPemFile() ? 'getPemFile' : $server->getPrivateKey() ? 'getPrivateKey' : false;
```

Comment:
This check was introduced by @koenvdheuvel - might want to check with him. I believe that issue here is caused by stacking ternary operator which is not recommended.